lizmap/www/js/edition.js

```diff
@@ -835,12 +826,11 @@ OpenLayers.Geometry.pointOnSegment = function(point, segment) {
                 if ( $(this).is(':checked') ) {
                     $('#edition-point-coord-x').attr('disabled','disabled');
                     $('#edition-point-coord-y').attr('disabled','disabled');
-                    if ( lizMap.controls.geolocation.layer.features.length != 0 ) {
-                        var geometryType = editionLayer.geometryType;
-                        var vertex = lizMap.controls.geolocation.layer.features[0].geometry;
-                        var px = editCtrls[geometryType].handler.layer.getViewPortPxFromLonLat({lon:vertex.x,lat:vertex.y});
-                        editCtrls[geometryType].handler.modifyFeature(px);
-                    }
+
+                    var geometryType = editionLayer.geometryType;
+                    var [lon, lat] = lizMap.mainLizmap.geolocation.getPositionInCRS(editionLayer['ol'].projection);
```

Comment:
Are you sure that the geolocation return position ?
Are you sure that the geolocation has started ?

Reply:
https://github.com/3liz/lizmap-web-client/pull/1450/commits/fd58c8ca7d5dc286e6a2d4d62e6e0c2d647d5af5